### DISCUSSION TITLE
No gpu detected, fallback to cpu. You can ignore this warning if this is intended.

### DISCUSSION THREAD

**Discussion by linhh29** [autogluon/autogluon#1693] [Q&A]
Hello!
I'm trying to use ImagePredictor in my machine, which has 1 3060ti,but the error denotes that it cannot  detect gpu.
Additionally, at the beginning of the output, it denotes that "The number of requested GPUs is greater than the number of available GPUs.Reduce the number to 1".
### The detail is as follow:

`time_limit=auto` set to `time_limit=7200`.
Reset labels to [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119]
Randomly split train_data into train[9199]/validation[1023] splits.
**The number of requested GPUs is greater than the number of available GPUs.Reduce the number to 1**
Starting fit without HPO
modified configs(<old> != <new>): {
root.img_cls.model   resnet101 != resnet50
root.misc.num_workers 4 != 16
root.misc.seed       42 != 327
root.train.early_stop_patience -1 != 10
root.train.early_stop_baseline 0.0 != -inf
root.train.early_stop_max_value 1.0 != inf
root.train.epochs    200 != 15
root.train.batch_size 32 != 16
}
Saved config to /mnt/d/Project/dog-breed-identification/e7616471/.trial_0/config.yaml
**No gpu detected, fallback to cpu. You can ignore this warning if this is intended.**
Model resnet50 created, param count:                                         23753912
AMP not enabled. Training in float32.
Disable EMA as it is not supported for now.
Start training from [Epoch 0]

Thanks in advance.

**Comment by Innixma** [comment]
Moved this to an issue: #1709
